src/main/java/baseball/Domain/NumberValidator.java

```diff
@@ -0,0 +1,34 @@
+package baseball.Domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+
+public class NumberValidator {
+
+	private static final int MAX = 9;
+	private static final int MIN = 1;
+
+	public void checkDuplicate(List<Integer> numbers){
+		Set<Integer> numberSet = new HashSet<>(numbers);
+		if(numberSet.size()!= numbers.size()) {
+			throw new IllegalArgumentException("숫자는 중복되면 안됩니다");
+		}
+	}
+
+	public void checkRange(List<Integer> numbers){
+		for(Integer a : numbers){
+			if(a < MIN || a > MAX){
```

Comment:
부등호 방향을 한쪽으로 일치시키면 다른 개발자가 연상하기가 쉬워요.
<img width="462" alt="image" src="https://github.com/PARK9898/java-baseball/assets/39221443/6cab632a-b1dd-49f9-b3e7-6604c0bae59a">

```suggestion
			if(a < MIN || MAX < a){
```

Reply:
* 사소하지만 꿀팁이네요! 적용시켰습니다